svm/src/account_loader.rs

```diff
@@ -2648,22 +2618,28 @@ mod tests {
                 &[],
                 vec![account_meta(program2, false)],
             )];
-            test_data_size_with_cache(
+            test_data_size(
                 ixns,
-                program_cache.clone(),
                 program2_size + upgradeable_loader_size + fee_payer_size,
             );
 
-            // programdata as instruction account double-counts it
+            // programdata as readonly instruction account double-counts it
```

Comment:
```suggestion
            // programdata as instruction account double-counts it
```

these tests are in a loop that use both read-only and writable account metas

Reply:
yep, but only one case is double counted. See `factor` below